extensions/websockets/client/runtime/src/main/java/io/quarkus/websockets/client/runtime/DisableLoggingAutoFeature.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.websockets.client.runtime;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.graalvm.nativeimage.hosted.Feature;
+
+import com.oracle.svm.core.annotate.AutomaticFeature;
+
+/**
+ * Disables logging during the analysis phase
+ */
+@AutomaticFeature
```

Comment:
Just keep in mind for the (not too distant) future, that GraalVM won't be doing classpath scanning to find these, so we'll have to add a build item and explicitly register them on the command line

Reply:
Good to know, thanks for the info! 👍🏻